FBTweak/FBTweakShakeWindow.m

```diff
@@ -86,6 +79,36 @@ - (void)_presentTweaks
   }
 }
 
+- (FBTweakViewController * _Nullable)findTweakViewController {
+  UIViewController *visibleViewController = self.rootViewController;
+  while (visibleViewController.presentedViewController != nil) {
+    visibleViewController = visibleViewController.presentedViewController;
+  }
+
+  if (![visibleViewController isKindOfClass:[FBTweakViewController class]]) {
+    return nil;
+  }
+
+  return (FBTweakViewController *)visibleViewController;
+}
+
+- (void)dismissTweaksViewController {
+  FBTweakViewController * _Nullable viewController = [self findTweakViewController];
```

Comment:
Did we tell that everything is nullable here, by default?
If so remove the _Nullable.

Reply:
added `NS_ASSUME_NOT_NULL` to the file and made everything more LT-like.